api/gateway/availability.go

```diff
@@ -34,7 +34,7 @@ func (h *Handler) handleHeightAvailabilityRequest(w http.ResponseWriter, r *http
 		writeError(w, http.StatusInternalServerError, heightAvailabilityEndpoint, err)
 		return
 	}
-	if headHeight := int(head.Height()); headHeight < height {
+	if headHeight := int(head.Height()); height >= headHeight+1 {
```

Comment:
Looking into the error that happened for the [Rollkit](https://github.com/rollkit/rollkit/pull/953#issuecomment-1562215384). They requested a height of 27, and the node head was 26

27 >= 26+1

So it will still error. 

Along with fixing this, I propose to reformat this to IMO more readable format:
`localHeight +1` < `height` (If `localHeight+1` is less than the requested `height` - error out.)

26+1 < 27 - does not hits the error case

Reply:
yes

Reply:
dream team ⭐